packages/dds/tree/src/test/util/summaryBenchmarkSizes.ts

```diff
@@ -0,0 +1,142 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { makeRandom } from "@fluid-internal/stochastic-test-utils";
+import { IsoBuffer } from "@fluidframework/common-utils";
+import { TransactionResult } from "../../checkout";
+import { singleTextCursor } from "../../feature-libraries";
+import { ISharedTree } from "../../shared-tree";
+import { detachedFieldAsKey, TreeValue } from "../../tree";
+import { brand } from "../../util";
+import { TestTreeProvider } from "../utils";
+
+const random = makeRandom(129384);
```

Comment:
This is stateful random number genertor, right? If we want our tests to be deterministic, each test should initialize its own random number generator as part of the test. Otherwise running tests in different orders, skipping tests (ex: with a test filter) or adding new tests will change what the other tests do.

Reply:
Added `seed` parameter in `getInsertsSummarySize( )` function so that each case is deterministic.